index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" media="(min-width:1000px)" href="css/general.css">
+    <link rel="stylesheet" media="(max-width:1000px)" href="css/style.css">
+    
+    <title>Portfolio</title>
+</head>
+<body>
+  <header class="main-section">
+    <section class="toolbar">
+      <nav class="desktop">
+        <div class="all-item"><a href="#">My Logo</a></div>
+        <div class="desk-item">
+          <div><a href="#portfolio">Portfolio</a></div>
+          <div><a href="#about">About</a></div>
+          <div><a href="#contact">Contact</a></div>
+        </div>
+        <div class="item menu"><a href="#" class="openMenu"><i class="fa fa-bars" aria-hidden="true"></i></a></div>
+      </nav>
+    </section>
+  </header>
```

Comment:
As requested by the previous reviewer the `logo` should have to be out of the `nav` tag and inside the `header` tag. Here I put the general structure of how content inside the `header`  tag should have to be 👇 

```
<header>
    <a>Log...</a>
    <nav>...</nav>
</header>

```